src/services/creategeojsonbloodhound.js

```diff
@@ -52,18 +53,28 @@ ngeo.CreateGeoJSONBloodhound;
  * @param {ol.proj.Projection=} opt_dataProjection Data projection.
  * @param {BloodhoundOptions=} opt_options optional Bloodhound options. If
  *     undefined, the default Bloodhound config will be used.
+ * @param {BloodhoundQueryOptions=} opt_query optional Bloodhound query options.
  * @return {Bloodhound} The Bloodhound object.
  */
 ngeo.createGeoJSONBloodhound = function(url, opt_filter, opt_featureProjection,
-    opt_dataProjection, opt_options) {
+    opt_dataProjection, opt_options, opt_query) {
```

Comment:
If you set `opt_query` and `opt_options` with  `{remote: {...} }` then opt_query will be ignored.
That can be complicated to understand. can you add a note about that in the documentation ?